packages/playwright/src/runner/uiMode.ts

```diff
@@ -128,6 +128,10 @@ class UIMode {
       port: options.port,
       persistentContextOptions: {
         handleSIGINT: false,
+        ...(isUnderTest() ? {
```

Comment:
drive-by: the UI Mode tests were not passing on my dark-mode configured mac with a wide display attached (chrome window was too small and theme mismatch which made CSS tests failing)